dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -6843,6 +6856,25 @@ else if(field.getFieldType().equals(Field.FieldType.DATE_TIME.toString())){
         }
     }
 
+    @NotNull
+    private String getUniqueFieldErrorMessage(final Field field, final Object fieldValue, final ContentletSearch contentletSearch) {
+
+        return String.format("Value of Field [%s] must be unique. Contents having the same value (%s): %s",
+                field.getVelocityVarName(), fieldValue, contentletSearch.getIdentifier());
+    }
+
+    public static void setUniquePerSite(final boolean uniquePerSite){
+        ESContentletAPIImpl.uniquePerSite = uniquePerSite;
+    }
+
+    public static boolean getUniquePerSite(){
+        return uniquePerSite;
```

Comment:
what happen if it is null?

Reply:
I turn it to Boolean 

https://github.com/dotCMS/core/pull/21843/commits/0396a20c99adfbe928755457c8631b48e9053ba8#diff-fa1ceaa19618a6b2bbc30e24c6f930b4971f417db50babb748c2e2837ba9eb82R6872